pkg/apis/migration/v1alpha1/migplan_types.go

```diff
@@ -144,3 +145,89 @@ func (r *MigPlan) GetRefResources(client k8sclient.Client) (*PlanResources, erro
 
 	return resources, nil
 }
+
+// PV Actions.
+const (
+	PvMoveAction = "move"
+	PvCopyAction = "copy"
+)
+
+// Name - The PV name.
+// StorageClass - The PV storage class name.
+// Action - The PV migration action (move|copy)
+// staged - A PV has been explicitly added/updated.
+type PersistentVolume struct {
+	Name         string `json:"name,omitempty"`
+	StorageClass string `json:"storageClass,omitempty"`
+	Action       string `json:"action,omitempty"`
+	staged       bool
+}
+
+// Collection of PVs
+type PersistentVolumes struct {
+	Content []PersistentVolume `json:"persistentVolumes"`
```

Comment:
Mainly to avoid the awkward redundant semantics of PersistentVolumes.PersistentVolumes.  By name, the struct is a collection.  The purpose of the struct field tags is specifically to support the names being different.

Reply:
Fair enough, I hadn't thought about that (re: purpose of struct field tags)